src/zenml/zen_stores/schemas/user_management_schemas.py

```diff
@@ -187,14 +188,33 @@ def to_model(self) -> TeamModel:
         )
 
 
+class RolePermissionsSchema(SQLModel, table=True):
+    """SQL Model for team assignments."""
+
+    permission_id: int = Field(primary_key=True, foreign_key="permissionschema.id")
```

Comment:
why is permission ID `int` not `UUID`?

Reply:
I would argue the permissions are maintained by us, the permissions_table could be seen as a db representation of an enum. I don't think uuid's would be necessary here